### DISCUSSION TITLE
My edu email does not allow Outside emails

### DISCUSSION THREAD

**Discussion by banana-man10** [community/community#51402] [GitHub Education]
Hello.
I am trying to apply for the edu perks but my email does not support outside emails outside of the district I made a support ticket and was asked to make. a discussion here Please tell me if github allows these types of edu emails. 
Thank you

**Comment by F0XN0T** [comment]
I understand your frustration with the difficulties you're facing in accessing the GitHub Student Developer Pack. It's concerning to hear that your email doesn't support outside emails outside of the district and that you haven't received a satisfactory response from GitHub despite creating a support ticket.

To answer your question, GitHub does support certain types of edu emails for accessing the student developer pack. However, it's important to check with GitHub's guidelines to make sure that your specific email address is eligible.

If you're still facing issues, one option could be to try reaching out to GitHub's customer support team again and explaining the situation in detail. It may also be helpful to connect with other affected students and creators to see if there are any collective actions that can be taken.

I hope this issue can be resolved soon, as it's affecting a large number of users and is urgent in nature.

**Reply by banana-man10** [reply]
Alright thank you. It's not like I need it anytime soon it's just good to have as I use GitHub alot for my projects and it would be needed for that stuff. My email is also eligible it's just the outside issue thing but other than that thank you! I will be contacting support to see if I can get my edu verified on this account
